src/3p-frame.js

```diff
@@ -257,14 +269,21 @@ export function getRandom(win) {
  * Otherwise null.
  * @param {!Window} parentWindow
  * @param {boolean=} opt_strictForUnitTest
+ * @param {string=} opt_type
+ * @param {boolean=} opt_disallowCustom whether 3p url should not use meta tag.
  * @return {?string}
  */
-function getCustomBootstrapBaseUrl(parentWindow, opt_strictForUnitTest) {
+function getCustomBootstrapBaseUrl(
+    parentWindow, opt_strictForUnitTest, opt_type, opt_disallowCustom) {
   const meta = parentWindow.document
       .querySelector('meta[name="amp-3p-iframe-src"]');
   if (!meta) {
     return null;
   }
+  if (opt_disallowCustom) {
```

Comment:
I can't because I want to warn if remote HTML is specified so either way I need the query selector